BBB_cci_mpf/sw/CMakeLists.txt

```diff
@@ -109,7 +109,7 @@ if (WIN32)
 endif(WIN32)
 
 
-include_directories(${PROJECT_SOURCE_DIR}/include)
```

Comment:
You should delete line 108, since your change on 112 makes it a duplicate.

Reply:
Done